covid-19.md

```diff
@@ -1,15 +1,26 @@
 ---
 layout: blog
 title: COVID-19 Updates
-date: 2021-12-08
+date: 2022-03-31
 ---
 
 We will keep this page updated with new information as it becomes available.
 
 Last updated: {{ page.date | date: "%e %B %Y" }}.
 
-## The Competition - SR2022
+## SR2022
 
-It is currently our intention to run an in person competition for the SR2022 competition cycle. This is of course dependent on the government guidelines which we are keeping a close eye on. If you have any COVID-19 specific questions about the competition please contact us at [{{ site.emails.teams }}][teams-email].
+Life is beginning to get back to normal, and this means we are finally able to do in person events again!
+However, we must recognise that some people still have to be cautious, to protect their own health or their friends or family.
+As such, we ask that everyone follows the below requests to help ensure we all keep safe, whilst still having lots of fun with robots!
+
+* If you are experiencing symptoms of COVID-19, or have tested positive for COVID-19 in the 7 days beforehand (or live with someone who has) please do not attend.
+* If you are able, please wear a mask when not in your team pit.
+* Please clean your hands regularly, using hand sanitizer or soap.
```

Comment:
Worth mentioning that we won't be providing sanitizer?

Reply:
I thought about this.

I'm largely still debating whether we need to provide some or not (at least for the main competition). However that's beyond the scope of this PR.

I can't say with any confidence whether or not the venue for either the techday or the competition will have hand sanitizer available anyway.
It is reasonable to assume the toilets will have some kind of soap or sanitizer in them, which means there is always the soap fallback.
Some young people can, at times, be incredibly pedantic and might use any kind of inaccuracy to justify not complying.
A number of people will carry their own anyway.
So it's easier to just not mention it.